index.html

```diff
@@ -178,7 +183,7 @@ <h3>CANOPY</h3>
         </div>
         <!-- card works  4-->
         <div class="work-container">
-          <div class="cards">
+          <div class="cards reverse-block">
             <div class="snapshot-portofolio">
               <img src="img/work-img/Snapshoot Portfolio 4.png" alt="SP" />
```

Comment:
- [ ] As requested by the previous reviewer please stick to the design as much as possible. I would suggest please make sure that image and content should occupy 50,50% means the page is divided half of it contains the project's picture and the other half contains the project's description. 
- [ ] I would suggest you please use `border: 1px solid #DFE1E6;` to your project to look similar to the Figma design.
- [ ] [Optional] I would suggest you there is more space from left and right of the page please update according to design.
### Your design
![his](https://user-images.githubusercontent.com/81727988/236620936-32a9bc73-6dd0-41c9-a47c-050bad7d4366.png)

### Fimga design
![fimga](https://user-images.githubusercontent.com/81727988/236620942-50e74342-3c04-4a41-bed7-9a2749a692f9.png)